v6/v6_interface.ml

```diff
@@ -20,17 +20,22 @@ let uri () = "file:" ^ !default_path
 
 type debug_info = string
 
+type edition_info = {
```

Comment:
The record introduces clarity - good. I would have called the type simply `edition` or `license` unless there is a type of that name already in use (outside this module). If this record provides details to an existing license, `license_attributes` or `license_attr` would be less generic than `info`. Anyway, I'm fine with keeping it.